common-util/src/main/java/org/sunbird/common/request/orgvalidator/OrgRequestValidator.java

```diff
@@ -74,4 +76,29 @@ public void validateUpdateOrgStatusRequest(Request request) {
           ERROR_CODE);
     }
   }
+
+  private void validateOrgLocationAndCode(Request orgRequest) {
+    Object locationIds = orgRequest.getRequest().get(JsonKey.LOCATION_ID);
```

Comment:
validateListParamWithPrefix -> use this - which will check data is of list type or not